lib/hcloud/server.rb

```diff
@@ -10,7 +10,11 @@ class Server
       created: :time,
       server_type: ServerType,
       datacenter: Datacenter,
-      image: Image
+      image: Image,
+      iso: Iso,
+      private_net: [Network],
+      public_net: Network,
```

Comment:
`public_net` is just a bunch of information and don't relate to any resource directly

Reply:
I dropped `public_net` again